velox/experimental/cudf/exec/ToCudf.cpp

```diff
@@ -219,4 +222,8 @@ bool cudfIsRegistered() {
   return isCudfRegistered;
 }
 
+bool isCudfOperator(const exec::Operator* op) {
+  return isAnyOf<NvtxHelper>(op);
+}
```

Comment:
You mention in your PR description that you want to 
>check it can be fully offloaded to cudf without extra data format conversion cost

But the cudf conversion operators also inherit from `NvtxHelper`. Did you mean to make a different function that checks that `CudfFromVelox` and `CudfToVelox` don't appear in the task?

Reply:
Partly yes, and I will check the replaced operators, I allow TableSan and ValueStream node with CudfFromVelox and CudfToVelox. ValueStream accepts shuffle reader input.